azkaban-common/src/main/java/azkaban/database/AbstractJdbcLoader.java

```diff
@@ -16,25 +16,31 @@
 
 package azkaban.database;
 
+import static azkaban.ServiceProvider.SERVICE_PROVIDER;
+
 import azkaban.metrics.CommonMetrics;
 import azkaban.utils.Props;
 import java.io.IOException;
 import java.sql.Connection;
+import javax.inject.Inject;
 import org.apache.commons.dbutils.DbUtils;
 import org.apache.commons.dbutils.QueryRunner;
 
 
 public abstract class AbstractJdbcLoader {
 
   private final AzkabanDataSource dataSource;
+  private final CommonMetrics commonMetrics;
 
-  public AbstractJdbcLoader(final Props props) {
+  @Inject
+  public AbstractJdbcLoader(final Props props, final CommonMetrics commonMetrics) {
     this.dataSource = DataSourceUtils.getDataSource(props);
```

Comment:
Not in this PR, but in the future, we should get datasource from Juice. 
@kunkun-tang